tests/api/admin/controller/test_library.py

```diff
@@ -210,7 +210,7 @@ def test_libraries_get_with_multiple_libraries(
                 FacetConstants.ORDER_TITLE,
                 FacetConstants.ORDER_AUTHOR,
             ] == settings_dict.get("facets_enabled_order")
-            assert ["French"] == settings_dict.get("large_collection_languages")
+            assert ["fre"] == settings_dict.get("large_collection_languages")
```

Comment:
It's not obvious why this has changed.

Reply:
It changed because the call to update settings on line 176 now validates the settings before saving, which normalizes the language codes. That normalization is tested elsewhere, so I think its fine to just update the test here.

Reply:
I think it's fine, too. I just didn't understand it.